ambry-frontend/src/main/java/com.github.ambry.frontend/FrontendMetrics.java

```diff
@@ -392,4 +386,50 @@ public FrontendMetrics(MetricRegistry metricRegistry) {
     putWithAccountAndContainerHeaderRate = metricRegistry.meter(
         MetricRegistry.name(AmbryBlobStorageService.class, "PutWithAccountAndContainerHeaderRate"));
   }
+
+  /**
+   * Class to hold different {@link RestRequestMetrics} for plain text, ssl, with and without encryption
+   */
+  class RestRequestMetricsGroup {
```

Comment:
`private static`

Reply:
it uses metricRegistry and hence can't be static. Will add private.

Reply:
Sorry. this is being access by AmbryBlobStorageService and hence can't make it private too